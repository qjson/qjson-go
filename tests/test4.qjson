v1 : 1h2m * 2
