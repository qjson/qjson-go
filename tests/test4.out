qjson: invalid numeric expression at line 1 col 11
